packages/react-core/src/components/Content/examples/ContentOrderedList.tsx

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
-import { TextList, TextListVariants, TextListItem } from '@patternfly/react-core';
+import { Content, ContentVariants } from '@patternfly/react-core';
 
 export const ContentOrderedList: React.FunctionComponent = () => (
-  <TextList component={TextListVariants.ol}>
-    <TextListItem>Donec blandit a lorem id convallis.</TextListItem>
-    <TextListItem>Cras gravida arcu at diam gravida gravida.</TextListItem>
-    <TextListItem>Integer in volutpat libero.</TextListItem>
-    <TextListItem>Donec a diam tellus.</TextListItem>
-    <TextListItem>Aenean nec tortor orci.</TextListItem>
-    <TextListItem>Quisque aliquam cursus urna, non bibendum massa viverra eget.</TextListItem>
-    <TextListItem>Vivamus maximus ultricies pulvinar.</TextListItem>
-  </TextList>
+  <Content component={ContentVariants.ol}>
+    <Content>Donec blandit a lorem id convallis.</Content>
```

Comment:
@tlabaj this example just needs to be updated so that the nested `<Content>`'s inside the `<Content component={ContentVariants.ol}>` have the `component={ContentVariants.li}` prop